src/pages/tutorials/smart-rollups/start-sandbox.md

```diff
@@ -0,0 +1,90 @@
+---
+id: start-sandbox
+title: "Part 2: Starting a Tezos sandbox"
```

Comment:
I think this ordering has changed from the original? Personally, I find the new order slightly confusing as you end up setting up a tezos sandbox, and then not using for several steps...

Reply:
Not sure if I follow. As it is now, the steps are:

6. Run `./sandbox.sh`
7. Leave the terminal instance running
8. Open a new terminal window
9. Enter the docker container
10. Run `octez-client rpc get /chains/main/blocks/head/metadata | grep protocol`

That octez-client command doesn't work without the sandbox running. So it seems to me that we use the sandbox right away. We have to have the sandbox running before running  the kernel in debug mode in part 3.

Reply:
As in, the current tutorial gets the octez distribution - and then does building/reducing size/debuggin.

Only then does it start the sandbox, immediately before deploying.

But the new section ordering starts the sandbox, in part 2. The sandbox isn't used until part 5.

Reply:
Oh, I didn't realize that the debugger didn't use the sandbox. I'll rearrange it.